### PR TITLE
Create model Place with rspec

### PR DESCRIPTION
**Pull request by ChernikovVolodymyr**
Create Place model
Create model rspec

### CODE REVIEW

File: db/migrate/20220304145225_create_places.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      t.integer :type
+      t.string :city
+      t.string :region
+      t.integer :rooms_available
+      t.integer :beds
+      t.integer :kids_beds
+      t.boolean :is_pets_allowed
+      t.integer :capacity, default: 2, null: false
+      t.integer :additional_child_place, default: 0
+      t.integer :coordinator
```
Comment (vova) [inline db/migrate/20220304145225_create_places.rb]: sorry, I didn't noticed that, but @s-nestor noticed, it'd be better to have coordinator_id

File: db/migrate/20220304145225_create_places.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
+class CreatePlaces < ActiveRecord::Migration[6.1]
+  def change
+    create_table :places do |t|
+      t.string :name, limit: 100, null: false
+      t.integer :type
```
Comment (sandra-nestor) [inline db/migrate/20220304145225_create_places.rb]: Also we have AccommodationType model, maybe change to `type_id`, too @vova??

File: db/migrate/20220304145225_create_places.rb (hunk 1 above)
Comment (ChernikovVolodymyr) [reply db/migrate/20220304145225_create_places.rb]: Done
Addressed here https://github.com/vova/war_shelter/pull/22/commits/7f8f34ffb8954478c8913c7b279be9aa00b3c4ef

File: db/migrate/20220304145225_create_places.rb (hunk 2 above)
Comment (vova) [reply db/migrate/20220304145225_create_places.rb]: u r right!

Comment (ChernikovVolodymyr) [reply db/migrate/20220304145225_create_places.rb]: Update with `accomodation_type_id`
Addressed her https://github.com/vova/war_shelter/pull/22/commits/35ffc92b4826db920f03eeb0ea3f500850cd430a

**Review by sandra-nestor — APPROVED** [review]

File: db/migrate/20220304145225_create_places.rb [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+      t.string :city
+      t.string :region
+      t.integer :rooms_available
+      t.integer :beds
+      t.integer :kids_beds
+      t.boolean :is_pets_allowed
+      t.integer :capacity, default: 2, null: false
+      t.integer :additional_child_place, default: 0
+      t.integer :coordinator_id
+      t.integer :status
```
Comment (oleksandr-kychun) [inline db/migrate/20220304145225_create_places.rb:15]: I believe we have PlaceStatus model, probably needs to be a foreign key
@vova is this correct?

Comment (vova) [reply db/migrate/20220304145225_create_places.rb:15]: yes, r u right!

**Pull request merged**